Parsers/Netscaler_parser.csl

```diff
@@ -0,0 +1,67 @@
+    SubjectName = extract(@'SubjectName\s\"\s(.*?)\"', 1, CitrixEventMessage),
+    CitrixSessionId = extract(@"SessionId\:\s(.*?)\s\-\s", 1, CitrixEventMessage),
+    CitrixUser = extract(@"User\s(.*?)\s", 1, CitrixEventMessage),
+    CitrixGroups = extract(@"Group\(s\)\s(.*?)\s\:\s", 1, CitrixEventMessage),
+    CitrixVserver_IP = extract(@"Vserver\s(.*?)\:\d+\s", 1, CitrixEventMessage),
+    Citrix_SrcIp = extract(@"Remote_ip\s(\d+[.]\d+[.]\d+[.]\d+)\s", 1, CitrixEventMessage),
+    Citrix_DestIp = extract(@"Destination\s(\d+[.]\d+[.]\d+[.]\d+)\s", 1, CitrixEventMessage),
+    Citrix_TotalBytesSend  = extract(@"Total_bytes_send\s(\d+)\s\-", 1, CitrixEventMessage),
+    Citrix_TotalBytesRecv = extract(@"Total_bytes_recv\s(\d+)\s\-", 1, CitrixEventMessage),
+    Citrix_AccessAction = extract(@"Access\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixCMD_Command = extract(@'Command\s\"(.*?)\"', 1, CitrixEventMessage),
+    CitrixCMD_Status = extract(@'Status\s\"(.*?)\"', 1, CitrixEventMessage),
+    CitrixVserver_Port = extract(@"Vserver\s{\d+[.]\d+[.]\d+[.]\d+}\:(.*?)\s\-\s", 1, CitrixEventMessage),
+    CitrixResource_URL = extract(@"\s\:\sGET\s(.*?)$", 1, CitrixEventMessage),
+    CitrixExtracted_GroupRAW = extract(@'Extracted\_groups\s\"(.*?)\"', 1, CitrixEventMessage),
+    CitrixContextRAW = extract(@"Context\s(.*?)\s", 1, CitrixEventMessage),
+    CitrixRequest_URL = extract(@"\srequest\:\s(.*?)\s", 1, CitrixEventMessage),
+    CitrixSession_StartTime = extract(@'Start_time\s\"(.*?)\"', 1, CitrixEventMessage),
+    Citrix_EndTime = extract(@'End_time\s\"(.*?)\"', 1, CitrixEventMessage),
+    CitrixSession_UserAgent = extract(@'Browser_type\s\"(.*?)\"', 1, CitrixEventMessage),
+    CitrixSession_ClientType = extract(@"SSLVPN_client_type\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixSession_FailedReason = extract(@'Failure_reason\s\"(.*?)\"', 1, CitrixEventMessage),
+    CitrixApp_ModulePath = extract(@"module_path\s(.*?)$", 1, CitrixEventMessage),
+    CitrixApp_Name = extract(@"app_name\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixApp_ProcId = extract(@"app_process_id\s(\d+)\s\-", 1, CitrixEventMessage),
+    CitrixApp_LaunchTime = extract(@"app_Launch_time\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixApp_Guid = extract(@"session_guid\s(.*?)\s\-", 1, CitrixEventMessage),
+    CitrixApp_DeviceSN = extract(@"device_serial_number\s(\d+)\s\-", 1, CitrixEventMessage),
+    CitrixSrvMon_IP = extract(@"MonServiceBinding_(.*?)\:", 1, CitrixEventMessage),
+    CitrixDvcMon_IP = extract(@"server_serviceGroup_NSSVC_HTTP_(.*?)\:", 1, CitrixEventMessage),
+    CitrixICA_SrcIP = extract(@"Source\s(.*?)\:", 1, CitrixEventMessage),
+    CitrixICA_SrcPort = extract(@"Source\s{\d+[.]\d+[.]\d+[.]\d+}\:(.*?)\s", 1, CitrixEventMessage),
+    CitrixICA_DestIP = extract(@"Destionation\s(.*?)\:", 1, CitrixEventMessage),
+    CitrixICA_DestPort = extract(@"Destionation\s{\d+[.]\d+[.]\d+[.]\d+}\:(.*?)\s", 1, CitrixEventMessage),
```

Comment:
This looks like typo for **_Destination_**. Can you check for line 60 and 61 ?

Reply:
Good Catch, correction made.